tools/perf/brave_page_sets/brave_loading_desktop.py

```diff
@@ -23,9 +23,7 @@ class DelayedSharedDesktopPageState(shared_page_state.SharedDesktopPageState):
   """ Brave version of SharedDesktopPageState with extras.
 
   1. A delay after the browser startup before running a test.
-  2. It closes extra tabs after finihsing the test to reuse the profile later
-     (used for rebasing profiles).
-  See shared_page_state.py for details.
+  2. It's used for rebasing profiles (do some setup in via the settings page).
```

Comment:
One of the prepositions is redundant (in or via?)

Reply:
thanks, fixed.